### DISCUSSION TITLE
[Bug]

### DISCUSSION THREAD

**Discussion by Hetox123** [ScoopInstaller/Scoop#4882] [Help and Questions] [answered]
#### Current Behavior
<!When installing scoop with the command iwr -useb get.scoop.sh | iex i get the following error:
Scoop requires 'C:\Windows\System32\Robocopy.exe' to work. Please make sure 'C:\Windows\System32' is in your PATH>

#### Expected Behavior
<!TO successfully install scoop>

### System details

**Windows version:** [10

**OS architecture:** 64bit

**PowerShell version:** [Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      19041  1620] i just pasted the output pls correct me because i know nothing regarding coding sorry

**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration
<!--Dont know what to write here >

```json
//# What do i do here i dont know what to write here


The output of $env:PATH.Split(';') : C:\Program Files\Java\jre1.8.0_311\bin

Once again sorry for not writing everything i have less to no knowledge about this 
kindly help me 
Thanks
```

**Comment by rashil2000** [comment]
What is the output of `$env:PATH.Split(';')`?

**Comment by Hetox123** [comment]
C:\Program Files\Java\jre1.8.0_311\bin
This is the output

**Comment by rashil2000** [comment]
Something is seriously wrong with your system Path. How come it only has Java's directory?

**Comment by Hetox123** [comment]
Then can you please help me
I really want to install scoop
Also can you message me on twitter GitHub comments are way scuffed and take a lot of time to message
I will surely put important info here but please 
My twitter is iHetox_

**Comment by rashil2000** [comment]
Open command prompt as administrator, and type this: 

```batch
setx /m PATH C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0;C:\Windows\System32\OpenSSH
```


After this, reinstall any programs that you have installed, and install Scoop.

**Comment by Hetox123** [comment]
This won't do anything to the data in my pc right? Like delete it or something?

**Comment by Hetox123** [comment]
Running the installer as administrator is disabled by default, see https://github.com/ScoopInstaller/Install#for-admin for details it says this now what do i do

**Comment by Hetox123** [comment]
' reinstall any programs that you have installed' means what programs specifically

**Comment by Hetox123** [comment]
AFter changing powershell execution policy it still says the same error 
Running the installer as administrator is disabled by default, see https://github.com/ScoopInstaller/Install#for-admin for details.

**Comment by rashil2000** [comment, marked as answer]
For installing Scoop, don't run PowerShell as administrator. Just run it normally.

**Comment by Hetox123** [comment]
Ok I'll try it when my pc is available

**Comment by Hetox123** [comment]
OK scoop is getting installed Thanks 
but now im trying to install scoop install git 7zip this and it tells that git is installed but when i try to Add bucket to it it says 
Git is required for buckets. Run 'scoop install git' and try again

And when installing git it says the following 

 scoop install git 7zip
Installing '7zip' (21.07) [64bit]
Loading 7z2107-x64.msi from cache
Checking hash of 7z2107-x64.msi ... ok.
Extracting 7z2107-x64.msi ... done.
Linking ~\scoop\apps\7zip\current => ~\scoop\apps\7zip\21.07
Creating shim for '7z'.
Copy-Item : Cannot find path 'C:\Users\hp\scoop\apps\scoop\current\supporting\shims\kiennq\shim.exe' because it does
not exist.
At C:\Users\hp\scoop\apps\scoop\current\lib\core.ps1:627 char:9
+         Copy-Item (get_shim_path) "$shim.exe" -Force
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\hp\sco...kiennq\shim.exe:String) [Copy-Item], ItemNotFoundExce
   ption
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.CopyItemCommand

Creating shortcut for 7-Zip (7zFM.exe)
Persisting Codecs
Persisting Formats
Running post-install script...
'7zip' (21.07) was installed successfully!
Notes
-----
Add 7-Zip as a context menu option by running: "C:\Users\hp\scoop\apps\7zip\current\install-context.reg"
Installing 'git' (2.35.3.windows.1) [64bit]
Loading PortableGit-2.35.3-64-bit.7z.exe from cache
Checking hash of PortableGit-2.35.3-64-bit.7z.exe ... ok.
Extracting dl.7z ... done.
Linking ~\scoop\apps\git\current => ~\scoop\apps\git\2.35.3.windows.1
Creating shim for 'git'.
Copy-Item : Cannot find path 'C:\Users\hp\scoop\apps\scoop\current\supporting\shims\kiennq\shim.exe' because it does
not exist.
At C:\Users\hp\scoop\apps\scoop\current\lib\core.ps1:627 char:9
+         Copy-Item (get_shim_path) "$shim.exe" -Force
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\hp\sco...kiennq\shim.exe:String) [Copy-Item], ItemNotFoundExce
   ption
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.CopyItemCommand

Creating shim for 'gitk'.
Copy-Item : Cannot find path 'C:\Users\hp\scoop\apps\scoop\current\supporting\shims\kiennq\shim.exe' because it does
not exist.
At C:\Users\hp\scoop\apps\scoop\current\lib\core.ps1:627 char:9
+         Copy-Item (get_shim_path) "$shim.exe" -Force
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\hp\sco...kiennq\shim.exe:String) [Copy-Item], ItemNotFoundExce
   ption
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.CopyItemCommand

Creating shim for 'git-gui'.
Copy-Item : Cannot find path 'C:\Users\hp\scoop\apps\scoop\current\supporting\shims\kiennq\shim.exe' because it does
not exist.
At C:\Users\hp\scoop\apps\scoop\current\lib\core.ps1:627 char:9
+         Copy-Item (get_shim_path) "$shim.exe" -Force
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\hp\sco...kiennq\shim.exe:String) [Copy-Item], ItemNotFoundExce
   ption
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.CopyItemCommand

Creating shim for 'tig'.
Copy-Item : Cannot find path 'C:\Users\hp\scoop\apps\scoop\current\supporting\shims\kiennq\shim.exe' because it does
not exist.
At C:\Users\hp\scoop\apps\scoop\current\lib\core.ps1:627 char:9
+         Copy-Item (get_shim_path) "$shim.exe" -Force
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\hp\sco...kiennq\shim.exe:String) [Copy-Item], ItemNotFoundExce
   ption
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.CopyItemCommand

Creating shim for 'git-bash'.
Copy-Item : Cannot find path 'C:\Users\hp\scoop\apps\scoop\current\supporting\shims\kiennq\shim.exe' because it does
not exist.
At C:\Users\hp\scoop\apps\scoop\current\lib\core.ps1:627 char:9
+         Copy-Item (get_shim_path) "$shim.exe" -Force
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\hp\sco...kiennq\shim.exe:String) [Copy-Item], ItemNotFoundExce
   ption
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.CopyItemCommand

Creating shortcut for Git Bash (git-bash.exe)
Creating shortcut for Git GUI (git-gui.exe)
'git' (2.35.3.windows.1) was installed successfully!
Notes
-----
Set Git Credential Manager Core by running: "git config --global credential.helper manager-core"

**Comment by rashil2000** [comment]
Your antivirus must have deleted the C:\Users\hp\scoop\apps\scoop\current\supporting\shims\kiennq\shim.exe file. You need to disable antivirus or add an exception for Scoop folder.
